Dashboard/app/js/lib/controllers/maps-controllers-common.js

```diff
@@ -129,6 +124,20 @@ FLOW.MapsController = Ember.ArrayController.extend({
         }
     },
 
+    clearMarker: function () {
```

Comment:
Whi is the function called clearMarker but then we also clear things from the sidebar?

Reply:
The primary purpose of the function is to clear the highlighting added after clicking on a point on the map. We also set the marker's associated properties to null so that the next point the user clicks starts on a clean slate